pkg/keccak/Makefile

```diff
@@ -0,0 +1,24 @@
+PKG_NAME    := keccak
+PKG_URL     := https://github.com/gvanas/KeccakCodePackage
+PKG_VERSION := bc592d1780a6b3204c26c6f872552ace8fc6fd65
+
+ifeq ($(CPU),native)
+	CPU_ARCHIVE_NAME := Native
```

Comment:
The 3 indented blocks should be 2 spaces and not tabs.

Reply:
Fixed.